classes/privacy/provider.php

```diff
@@ -39,7 +39,7 @@ class provider implements null_provider {
      *
      * @return  string
      */
-    public static function _get_reason() {
+    public static function get_reason() : string {
```

Comment:
Not sure if we should have strict type decleration here if we want to support older moodles which could run on PHP < 7?

Reply:
Fixed here: https://github.com/catalyst/moodle-local_aws/blob/update_to_3.208.1/classes/privacy/provider.php#L42 :)

Reply:
@jwalits, I have reverted to _get_reason(). That way this will pass unit tests on both old and new PHP versions. I know it's not ideal, if this PR is approved then I'm more than happy to split it into 2 versions :)